app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    names = command.split()
+    cd_type, first_file, copys_file = names
```

Comment:
You should check the length of `names` before unpacking, otherwise, you could get an error